pkg/apis/cluster/v1alpha1/cluster_utils_test.go

```diff
@@ -25,13 +25,6 @@ func TestString(t *testing.T) {
 			},
 			want: clusterName,
 		},
-		{
-			name: "%v variable test",
-			fmtFunc: func() string {
-				return fmt.Sprintf("%v", *cluster1)
-			},
-			want: "{{ } {cluster1      0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] []  []} {  <nil> false     []} { [] [] <nil> <nil>}}",
-		},
```

Comment:
Removed this test because it is very hard to maintain. The maintain effort might > it's benefit.